src/mapper/src/lib/components/page/legend.svelte

```diff
@@ -0,0 +1,48 @@
+<script lang="ts">
+	import { clickOutside } from '../../../utilFunctions/clickOutside.ts';
```

Comment:
It's possible to use aliases here to avoid relative imports.

Svelte has a convention of using `$`. Currently we have `$lib` linking to `src/lib` and `$components` linking to `src/components`.

I'll merge this PR and we can update it later, perhaps we simply have a `$src` linking to `src` root dir to simplify things 😄

Reply:
sure!